src/main/java/jp/ac/keio/bio/fun/xitosbml/cli/README.md

```diff
@@ -1 +1,41 @@
-for cli version.
+# XitoSBML_CLI
+
+- Command Line Interface version for XitoSBML. 
+- Allows the use of XitoSBML API from the command line to obtain spatial SBML models from the segmentations of microscopic cell images.
+- Uses [Picocli](https://picocli.info/) as the command line parser which is a modern framework for building powerful, user-friendly, GraalVM-enabled command line applications.
+- Currently only supports 'Cytoplasm' as the default cellular domain.
+
+
+## How to Use
+
+Instructions on: 
+
+### Building
+```sh
+% git clone https://github.com/Medha-B/XitoSBML.git
```

Comment:
Maybe add 
`cd XitoSBML`

Reply:
Added cd XitoSBML. May I merge this now?